src/main/java/goodgenerator/loader/ComponentAssemblyLineRecipeLoader.java

```diff
@@ -363,4 +399,44 @@ private static List<ItemStack> getWrappedCircuits(ItemStack item, int total, Str
 
         return stacks;
     }
+
+    private static List<ItemStack> getMagnetoConversion(ItemStack item, int total) {
+        ArrayList<ItemStack> stacks = new ArrayList<>();
+        ItemData data = GT_OreDictUnificator.getAssociation(item);
+        if (data != null) {
+            if (total >= 64) {
+                double multiplier = magnetoConversionMultipliers.get(data.mPrefix);
+                stacks.addAll(
+                        getWrappedCircuits(
+                                GT_OreDictUnificator.get(OrePrefixes.circuit, Materials.Infinite, 1),
+                                (int) (total * multiplier),
+                                "circuitInfinite"));
+            }
+            stacks.addAll(multiplyAndSplitIntoStacks(item, total));
+        }
+        return stacks;
+    }
+
+    private static void addEternity(ArrayList<FluidStack> fluidInputs) {
```

Comment:
If this does both Eternity and mhdcsm it should say. Maybe change the method name?

Reply:
this adds eternity based on the amount of mhdcsm present

Reply:
AHHH. haven't played in a while and I forgot.